boot/bootutil/src/loader.c

```diff
@@ -781,9 +782,21 @@ boot_validate_slot(struct boot_loader_state *state, int slot,
 #if defined(MCUBOOT_OVERWRITE_ONLY) && defined(MCUBOOT_DOWNGRADE_PREVENTION)
     if (slot != BOOT_PRIMARY_SLOT) {
         /* Check if version of secondary slot is sufficient */
-        rc = boot_version_cmp(
-                &boot_img_hdr(state, BOOT_SECONDARY_SLOT)->ih_ver,
-                &boot_img_hdr(state, BOOT_PRIMARY_SLOT)->ih_ver);
+
+	/* Prototype to be moved in to hook */
+	#if defined(CONFIG_SOC_NRF5340_CPUAPP)
+	if (BOOT_CUUR_IMG(state) == 1) {
```

Comment:
Have you actually tried this?

Reply:
Yeah reverted a change but added typo sorry.